packages/@aws-cdk/aws-certificatemanager/lib/certificate.ts

```diff
@@ -72,6 +72,13 @@ export interface CertificateProps {
    * @default CertificateValidation.fromEmail()
    */
   readonly validation?: CertificateValidation;
+
+  /**
+   * Enable or disable transparency logging for this certificate
```

Comment:
Can you add something here about how enabling/disabling this works? For example I think you can't change it after creation?

Reply:
Well you technically can change it in the api, however it will not have an effect. When a certificate is registered/logged it cannot be undone.